addons/point_of_sale/static/src/js/models.js

```diff
@@ -457,7 +457,7 @@ class PosGlobalState extends PosModel {
         const products = await this.env.services.rpc({
             model: 'pos.session',
```

Comment:
maybe pass "active_test=true" in the context instead

Reply:
Thanks for your feedback.
Will the result change? I want to learn it.
Thanks,

Reply:
It should not change the result, it is just that the ORM is adding "active=true" in the requests when active is not passed.

So instead of having a sql request that mention active in (true,false), it will just not mention 'active' in the select and so return all of them

Reply:
Thanks,
I changed it. But it works when it's "active_test=false".